### DISCUSSION TITLE
**TOOL** List Converter (URL)

### DISCUSSION THREAD

**Discussion by Deadlineem** [YimMenu/HorseMenu#183] [General]
Original was removed by me, i tossed that website.

If youre looking for something like this, check out

https://github.com/Deadlineem/RDR2-Tools
OR
use them directly at https://kaos.bot.nu/rdr2

Q: This is just a native database, why use this when there are plenty of others?
A: This allows you to edit native parameters on the site itself and outputs the complete `namespace::native(param)` combination which you can simply paste into the Script Generator and create your custom features easily without Visual Studio

Screenshots:

<img width="1918" height="989" alt="image" src="https://github.com/user-attachments/assets/92c9cf98-5aae-4861-8fb0-06b178ced0e0" />
<img width="1079" height="595" alt="image" src="https://github.com/user-attachments/assets/03a27614-2e28-4d8c-9740-405c67290b4c" />
<img width="1088" height="275" alt="image" src="https://github.com/user-attachments/assets/cd7dac80-2c64-4c07-a06e-7ed0acbe115c" />

<img width="1919" height="987" alt="image" src="https://github.com/user-attachments/assets/11203933-bd16-4a23-a858-a330dc2808ad" />
<img width="1042" height="880" alt="image" src="https://github.com/user-attachments/assets/090c49d5-3db9-441e-8517-f7a7b2a51fd1" />

**Comment by Rxann** [comment]
Looks good! Will definitely use in the future! Thanks!

**Comment by jwrist71** [comment]
...and it's [ http://bedrock.root.sx/objlist.php ] been taken down

**Reply by Deadlineem** [reply]
https://kaos.bot.nu/rdr2/converter.php <-- New version of the list converter
https://kaos.bot.nu/rdr2/ <-- Native DB
https://kaos.bot.nu/rdr2/creator.php <-- Script Generator (HorseMenu/YimMenu)

See the Original Post to see the updated info on these.

**Reply by jwrist71** [reply]
that works. thanks for replying as well

**Reply by Deadlineem** [reply]
> that works. thanks for replying as well

You're welcome, I get emails when comments come through on github so I try to check them as soon as i can.
